packages/certs-v5/test/commands/certs/auto/enable.--wait.js

```diff
@@ -0,0 +1,168 @@
+'use strict'
+/* globals describe it beforeEach cli */
+
+let expect = require('chai').expect
+let nock = require('nock')
+let certs = require('../../../../commands/certs/auto/enable.js')
+const sinon = require('sinon')
+const lolex = require('lolex')
+
+describe('heroku certs:auto:enable --wait', function () {
+  let clock
+  let sandbox
+
+  beforeEach(function () {
+    cli.mockConsole()
+    nock.cleanAll()
+    sandbox = sinon.sandbox.create()
+    clock = lolex.install()
+    clock.setTimeout = function (fn, timeout) { fn() }
+  })
+
+  afterEach(function () {
+    clock.uninstall()
+    sandbox.restore()
+  })
+
+  it('waits until all certs are issued and notifies', () => {
+    const notifySpy = sandbox.spy(require('@heroku-cli/notifications'), 'notify')
+
+    let acmApi = nock('https://api.heroku.com', {
+      reqheaders: { 'Accept': 'application/vnd.heroku+json; version=3.cedar-acm' }
+    })
+    acmApi.post('/apps/example/acm', {}).reply(200, { acm: true })
+
+    let domainsApi = nock('https://api.heroku.com')
+    domainsApi.get('/apps/example/domains').reply(200, [
+      { 'kind': 'heroku', 'hostname': 'tokyo-1050.herokuapp.com', 'cname': null },
+      { 'kind': 'custom', 'hostname': 'heroku-acm.heroku-cli-sni-test.com', 'cname': 'heroku-acm.heroku-cli-sni-test.com.herokudns.com'},
+      { 'kind': 'custom', 'hostname': 'heroku-san-test.heroku-cli-sni-test.com', 'cname': 'heroku-san-test.heroku-cli-sni-test.com.herokudns.com'},
+    ])
+    let now = new Date().toISOString()
+    domainsApi.get('/apps/example/domains').reply(200, [
+      { 'kind': 'heroku', 'hostname': 'tokyo-1050.herokuapp.com', 'cname': null },
+      { 'kind': 'custom', 'hostname': 'heroku-acm.heroku-cli-sni-test.com', 'cname': 'heroku-acm.heroku-cli-sni-test.com.herokudns.com', 'acm_status': 'in-progress', 'updated_at': now },
+      { 'kind': 'custom', 'hostname': 'heroku-san-test.heroku-cli-sni-test.com', 'cname': 'heroku-san-test.heroku-cli-sni-test.com.herokudns.com', 'acm_status': 'in-progress', 'updated_at': now },
+    ])
+    domainsApi.get('/apps/example/domains').reply(200, [
+      { 'kind': 'heroku', 'hostname': 'tokyo-1050.herokuapp.com', 'cname': null },
+      { 'kind': 'custom', 'hostname': 'heroku-acm.heroku-cli-sni-test.com', 'cname': 'heroku-acm.heroku-cli-sni-test.com.herokudns.com', 'acm_status': 'dns-verified', 'updated_at': now },
+      { 'kind': 'custom', 'hostname': 'heroku-san-test.heroku-cli-sni-test.com', 'cname': 'heroku-san-test.heroku-cli-sni-test.com.herokudns.com', 'acm_status': 'failing', 'updated_at': now },
```

Comment:
I guess this didn't affect the test, but here the second domains fails and in the next answer the certificate is issued. I don't know apps/xxx/domains, but sounds weird

Reply:
Yeah, that's because "Failing" is an intermediate state according to [this](https://devcenter.heroku.com/articles/automated-certificate-management#view-your-certificate-status); it can go from failing to either Cert Issued or Failed.

Maybe I can clarify that on the tests

Reply:
I've added some comments to address this, hope that clarifies this.